be/src/column/chunk.cpp

```diff
@@ -302,12 +302,12 @@ std::string Chunk::debug_row(uint32_t index) const {
     return os.str();
 }
 
-void Chunk::merge(Chunk* src) {
-    DCHECK_EQ(src->num_rows(), num_rows());
-    for (auto& it : src->_slot_id_to_index) {
+void Chunk::merge(Chunk&& src) {
```

Comment:
Why use a right-value reference in this place?

Reply:
Because `merge` is like move semantics: move columns in `src` chunk to this chunk.